src/Model/Recipe.php

```diff
@@ -190,7 +190,15 @@ public function __construct(
 
         public ?string $behatWwwRoot = null,
 
-        public ?string $lang = null
+        public ?string $lang = null,
+
+        public ?string $configFile = null,
+
+        public $config = null,
```

Comment:
We should create a Config.php Model in the Model folder.
That way we can put all the fields we want to support into it.
Then this can be

public ?Config $config = null,